doc/source/glossary.rst

```diff
@@ -50,19 +142,101 @@ Glossary
 
        Fast element-wise operations, called a :term:`ufunc`, operate on arrays.
 
+
    array_like
        Any sequence that can be interpreted as an ndarray.  This includes
        nested lists, tuples, scalars and existing arrays.
 
+
+   array scalar
+       For uniformity in handling operands, NumPy treats
+       a :doc:`scalar <reference/arrays.scalars>` as an array of zero
+       dimension.
+
+
+   attribute
+       A property of an object that can be accessed using ``obj.attribute``,
+       e.g., ``shape`` is an attribute of an array::
+
+         >>> x = np.array([1, 2, 3])
+         >>> x.shape
+         (3,)
```

Comment:
This is also present in the python glossary (`` :term:`python:attribute` ``). Should we omit it?

Reply:
Absolutely, yes, omit. This is the content I wanted to get rid of. I don't know why it shows as an addition.